php-exercise-1/index.php

```diff
@@ -0,0 +1,5 @@
+<?php
+include 'Controller/helloController.php';
```

Comment:
Please apply MVC pattern on this. It is stated that view should be returned to the user as it is the interface that presents and accepts information from the users.

Reply:
I modified the code and the view is now returned instead of the controller.